README.md

```diff
@@ -17,8 +17,57 @@ tabs.select(1) //change to the "baz" tab.
 
 document.body.appendChild(tabs)
 
+setTimeout(
+  function () { tabs.select(0) },
+  2000
+)
 ```
 
+## Styling
+
+Hypertabs classes take inspiration from [micro-css](https://github.com/mmckegg/micro-css) where styling is super tightly specified using `>` - direct child only to ensure your styling don't leak.
+
+In nesting, the box schema looks like this :
+
+```
```

Comment:
**BREAKING CHANGE**
Read this to see the layout that gets built.
I dropped more official BEM in favour of direct-children style styles. Happy to discuss if it's not gonna serve.